### ISSUE TITLE
Firehose http endpoint destination not being returned

### ISSUE THREAD

**Issue by joro550** [localstack/localstack#3384]
# This is a ...

- [x] bug report
- [ ] feature request

# Detailed description
I'm attempting to setup a fire hose kinesis stream that delivers messaeges to a http endpoint, using the c# sdk here's my code:

```
var client = new AmazonKinesisFirehoseClient(new AmazonKinesisFirehoseConfig
{
    ServiceURL = "http://localhost:4566"
});

var r = await client.CreateDeliveryStreamAsync(new CreateDeliveryStreamRequest
{
    DeliveryStreamName = "stream",
    DeliveryStreamType = DeliveryStreamType.DirectPut,
    HttpEndpointDestinationConfiguration = new HttpEndpointDestinationConfiguration
    {
        EndpointConfiguration = new HttpEndpointConfiguration
        {
            Url = "https://localhost:5000/hello",
            Name = "Local"
        }
    }
});

var description = await client.DescribeDeliveryStreamAsync(new DescribeDeliveryStreamRequest
    {DeliveryStreamName = "stream"});

var destinations = description.DeliveryStreamDescription.Destinations.ToList();
```

## Expected behavior
Destinations to return a http endpoint configuration

## Actual behavior
No destinations are returned

# Steps to reproduce
All in description

## Command used to start LocalStack
docker-compose up -d

docker-compose file:
```
version: "3"

services:
  localstack:
    image: localstack/localstack:latest
    ports:
      - "4566:4566"
      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
    environment:
      - DOCKER_HOST=unix:///var/run/docker.sock
    volumes:
      - "/var/run/docker.sock:/var/run/docker.sock"
      - "/private${TMPDIR}:/tmp/localstack"
 ```

**Comment by whummer** [comment]
Thanks for reporting @joro550 . This should be fixed in #3825 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks!

**Comment by joro550** [comment]
Looks like this now gives me a destination, thanks!
